doc/book/cookbook/setting-locale-depending-routing-parameter.md

```diff
@@ -0,0 +1,303 @@
+# How can I setup the locale depending on a routing parameter?
+
+It is a common task to have an localized web application where the setting of
+the locale (and therefor the language) depends on the routing. There are other 
+ways of setting a language like using the session or a specialized sub-domain.
+In this recipe we will concentrate on using the a routing parameter.
+
+## Setting up the route ##
+
+If you want to set the locale depending on an routing parameter, you first have
+to add a language parameter to your routes. In this example we use the `lang` 
```

Comment:
@weierophinney 

I added a new PR #244 with an additional recipe. To have both variants in one recipe would bloat it. I hope this is ok.